cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -53,9 +53,9 @@ cannot connect to the requested virtual machine. On the client, update the
 live migration. As a workaround, use an additional `multus`-backed network.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1693532[*BZ#1693532*])
 
-* {ProductName} cannot reliably identify node drains that are triggered by
+* {CNVProductNameStart} cannot reliably identify node drains that are triggered by
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
-the nodes of any clusters where {ProductName} is deployed. The nodes might not
+the nodes of any clusters where {CNVProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
 The current solution is to put nodes into maintenance.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
```

Comment:
Not an issue with this line, but on line 69 there is an instance of `CNV` that should be replaced with `{CNVProductNameStart}`